packages/mui-material/src/CssBaseline/CssBaseline.js

```diff
@@ -43,6 +43,10 @@ export const styles = (theme, enableColorScheme = false) => {
     'strong, b': {
       fontWeight: theme.typography.fontWeightBold,
     },
+    a: {
+      color:
+        theme.palette.mode === 'dark' ? theme.palette.primary[300] : theme.palette.primary[700],
```

Comment:
Regardless of which color we choose, we should also add the support for theme.vars as well. I would recommend adding this value as a palette key and use that directly instead of hardcoding the values in the `CssBaseline`.

Reply:
@mnajdova Can I ask you to double-check the last commit to see if I've done that correctly? 😬